pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["scikit-build-core[pyproject]>=0.3.3"]
```

Comment:
Why is scikit required?

Reply:
Scikit handles the cmake to python integration. Even though it's weirdly named [scikit-build-core](https://github.com/scikit-build/scikit-build-core) is a build tool and not a scikit library.